.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @exu @nicufk @vsukhin @vLia @povilasv @dejanzele @haneabogdan @rangoo94 @emil2k @WitoDelnat @olensmar
```

Comment:
not important: missing EOL, and probably we can use group instead of specific names